src/views/SystemIntake/Page1/index.tsx

```diff
@@ -0,0 +1,218 @@
+import React, { useState } from 'react';
+import { Field, FieldArray, FormikProps } from 'formik';
+import TextField from 'components/shared/TextField';
+import CheckboxField from 'components/shared/CheckboxField';
+import { DropdownField, DropdownItem } from 'components/shared/DropdownField';
+import cmsDivisionsAndOffices from 'constants/enums/cmsDivisionsAndOffices';
+import cmsGovernanceTeams from 'constants/enums/cmsGovernanceTeams';
+import { SystemIntakeForm } from 'types/systemIntake';
+
+type Page1Props = {
+  formikProps: FormikProps<SystemIntakeForm>;
+};
+
+const Page1 = ({ formikProps }: Page1Props) => {
+  const [isReqAndBusOwnerSame, setSameness] = useState(false);
+  return (
+    <>
+      <Field
+        as={TextField}
+        id="IntakeForm-Name"
+        label="Name"
+        maxLength={50}
+        name="name"
+      />
+      <div className="width-card">
+        <Field
+          as={TextField}
+          id="IntakeForm-Acronym"
+          label="Acronym"
+          maxLength={10}
+          name="acronym"
+        />
+      </div>
+      <Field
+        as={TextField}
+        id="IntakeForm-Requestor"
+        label="Requestor"
+        maxLength={50}
+        name="requestor"
+        onChange={(e: any) => {
+          if (isReqAndBusOwnerSame) {
```

Comment:
This is the first instance where we are hijacking the Formik `onChange`.

By default, Formik updates the field based on the `name` and `value` of the field. IF the requestor and business owner are the same, we need to update BOTH fields.

That is what we are doing here.